### PR TITLE
3.0/feature/accordion

### PR DESCRIPTION
**Pull request by nRamstedt**
Refactor script-based guide module using styleguide v2 components.

**Changelog**
- Replace card implementation with accordion from styleguide
- Refactor Prev/Next Button logic 
- Refactor mailing todo list feature
- Fix issues with Todo modal & z-index
- Add loader to email form
- Add notices to email form
- Render recaptcha using javascript

**Known bug/limitations**
- Does not lock next/prev button based on checked options
- Required checkbox feature not re-implemented
- Step 1 should be open initially but requires changes in component library and should be fixed outside of this PR

**How to test**
1. Build assets using npm run build
2. install & activate wordpress plugin
3. Enable module in modularity settings
4. Create a new guide
5. Test checkbox (should show different content & populate todolist)
6. Test pagination (should paginate when clicking next/prev button)
7. Test todo checklist email feature

### CODE REVIEW

File: templates/guide.blade.php [hunk 1]
```diff
@@ -1,108 +1,55 @@
...
+                @accordion__item([
+                    'heading' => $loop->iteration . '. '  . $step['title'],
+                    'classList' => ['js-modularity-guide__section'],
+                    'attributeList' => ['data-guide-step' => $loop->iteration]
+                ])
+
+                    @if (isset($step['content']) && !empty($step['content']))
+                        @foreach ($step['content'] as $content)
+                            @include('partials.' . $content['acf_fc_layout'], array('stepId' => $j))
+                            @php $j++; @endphp
```
Comment (sebastianthulin) [inline templates/guide.blade.php:18]: Use magic variable $loop->count

File: templates/partials/todo.blade.php [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
-
-        <table class="table mod-guide-todo-list">
+<div class="o-grid modularity-guide-todos js-modularity-guide-todos">
+    <div class="o-grid-12">
+        <table class="table mod-guide-todo-list js-modularity-guide-todos__table">
             <thead>
                 <tr>
-                    <th class="text-center print-only" width="20">{{_e('Done', 'modularity-guides')}}</th>
                     <th>{{_e('Title', 'modularity-guides')}}</th>
                     <th>{{_e('Link', 'modularity-guides')}}</th>
```
Comment (sebastianthulin) [inline templates/partials/todo.blade.php:7]: Is this translatable without <?php tags? Previously po-edit haven't been able to find these.

File: source/js/Content/Todo.js [hunk 3]
```diff
@@ -1,51 +1,194 @@
...
+            RECAPTCHA_WIDGETS[`GreCaptchaWidget-${index}`] = grecaptcha.render(recaptchaContainer, {
+                sitekey: recaptchaSiteKey,
+                callback: handleRecaptcha,
+                'expired-callback': handleRecaptcha,
+            });
+        }
+    }
+
+    /**
+     * Query ToDo sections & initialize great things
```
Comment (sebastianthulin) [inline source/js/Content/Todo.js:181]: 😂

File: templates/guide.blade.php (hunk 1 above)
Comment (nRamstedt) [reply templates/guide.blade.php:18]: Fixing old code structure is not within the scope of this PR.

File: templates/partials/todo.blade.php (hunk 2 above)
Comment (nRamstedt) [reply templates/partials/todo.blade.php:7]: Old code, I have only modified what did not work. Haven't test this one out though

File: source/js/Content/Todo.js (hunk 3 above)
Comment (nRamstedt) [reply source/js/Content/Todo.js:181]: lets make modularity guide great again

File: templates/partials/todo.blade.php (hunk 2 above)
Comment (nRamstedt) [reply templates/partials/todo.blade.php:7]: Tested & verified - it works.

**Pull request merged**